docs/shared/features/powerpack/conformance.md

```diff
@@ -52,29 +52,56 @@ Conformance rules are configured in the `conformance` property of the `nx.json`
 }
 ```
 
-## Enforce Rules with the `nx conformance` Command
+## Enforce Rules with the `nx conformance` and `nx conformance:check` commands
 
-The `@nx/powerpack-conformance` plugin enables the `nx conformance` command which checks all the configured rules. This command should be added to the beginning of your CI process so that the conformance rules are enforced for every PR.
+The `@nx/powerpack-conformance` plugin enables the `nx conformance` and `nx conformance:check` commands which checks all the configured rules. The difference is that `nx conformance` will invoke any fix generators on a configured rule automatically, whereas `nx conformance:check` will only check the current workspace state and provide violations if applicable.
+
+Therefore, `nx conformance` is intended to be run locally while working on a feature branch, be added to the beginning of your CI process so that the conformance rules are enforced for every PR.
```

Comment:
```suggestion
Therefore, `nx conformance` is intended to be run locally while working on a feature branch. `nx conformance:check` should be added to the beginning of your CI process so that the conformance rules are enforced for every PR.
```